pkg/cmd/package/nuget/create/create.go

```diff
@@ -0,0 +1,314 @@
+	flags.BoolVar(&packFlags.Verbose.Value, packFlags.Verbose.Name, false, "Verbose output.")
+	flags.BoolVar(&packFlags.Overwrite.Value, packFlags.Overwrite.Name, false, "Allow an existing package file of the same ID/version to be overwritten.")
+	flags.StringSliceVar(&createFlags.Author.Value, createFlags.Author.Name, []string{}, "Add author/s to the package metadata.")
+	flags.StringVar(&createFlags.Title.Value, createFlags.Title.Name, "", "The title of the package.")
+	flags.StringVar(&createFlags.Description.Value, createFlags.Description.Name, "", "A description of the package, defaults to \"A deployment package created from files on disk.\".")
+	flags.StringVar(&createFlags.ReleaseNotes.Value, createFlags.ReleaseNotes.Name, "", "Release notes for this version of the package.")
+	flags.StringVar(&createFlags.ReleaseNotesFile.Value, createFlags.ReleaseNotesFile.Name, "", "A file containing release notes for this version of the package.")
+	flags.SortFlags = false
+
+	return cmd
+}
+
+func createRun(opts *NuPkgCreateOptions) error {
+	if !opts.NoPrompt {
+		if err := pack.PackageCreatePromptMissing(opts.PackageCreateOptions); err != nil {
+			return err
+		}
+		if err := PromptMissing(opts); err != nil {
+			return err
+		}
+	}
+
+	if opts.Id.Value == "" {
+		return errors.New("must supply a package ID")
+	}
+
+	err := applyDefaultsToUnspecifiedPackageOptions(opts)
+	if err != nil {
+		return err
+	}
+
+	nuspecFilePath := ""
+	if shouldGenerateNuSpec(opts) {
+		nuspecFilePath, err = GenerateNuSpec(opts)
+		if err != nil {
+			return err
+		}
+		opts.Include.Value = append(opts.Include.Value, opts.Id.Value+".nuspec")
+	}
+
+	pack.VerboseOut(opts.Verbose.Value, "Packing \"%s\" version \"%s\"...\n", opts.Id.Value, opts.Version.Value)
+	outFilePath := pack.BuildOutFileName("nupkg", opts.Id.Value, opts.Version.Value)
+
+	err = pack.BuildPackage(opts.PackageCreateOptions, outFilePath)
+	if err != nil {
+		return err
+	}
+
+	if !opts.NoPrompt {
+		autoCmd := flag.GenerateAutomationCmd(
+			opts.CmdPath,
+			opts.Author,
+			opts.Title,
+			opts.Description,
+			opts.ReleaseNotes,
+			opts.ReleaseNotesFile,
+			opts.Id,
+			opts.Version,
+			opts.BasePath,
+			opts.OutFolder,
+			opts.Include,
+			opts.Verbose,
+			opts.Overwrite,
+		)
+		fmt.Fprintf(opts.Writer, "\nAutomation Command: %s\n", autoCmd)
+	}
+
+	if nuspecFilePath != "" {
+		return os.Remove(nuspecFilePath)
+	}
+
+	return nil
+}
+
+func PromptMissing(opts *NuPkgCreateOptions) error {
+	if len(opts.Author.Value) == 0 {
+		message := "Author"
+		for {
+			var author string
+			if err := opts.Ask(&survey.Input{
+				Message: message,
+				Help:    "Add an author to the package metadata.",
+			}, &author); err != nil {
+				return err
+			}
+			if author == "" {
+				break
+			}
+			opts.Author.Value = append(opts.Author.Value, author)
+			message = message + " (leave blank to continue)"
```

Comment:
This looks like it will keep asking for more and more authors until you finally enter an empty line, which is cool.

This string concatenation looks like the output will be 

```
Author:
Author (leave blank to continue):
Author (leave blank to continue) (leave blank to continue):
Author (leave blank to continue) (leave blank to continue) (leave blank to continue):
```

Also - it looks like author is optional? If it is, maybe we should put the "leave blank" bit up there on the first prompt as well.